acs-integration-tests/pom.xml

```diff
@@ -642,22 +629,8 @@
                             <parallel>methods</parallel>
                             <argLine></argLine>
                             <systemPropertyVariables>
-                                <ACS_DEFAULT_ISSUER_ID>${ACS_DEFAULT_ISSUER_ID}</ACS_DEFAULT_ISSUER_ID>
-                                <ACS_SERVICE_ID>predix-acs</ACS_SERVICE_ID>
-                                <ACS_URL>${ACS_URL}</ACS_URL>
-                                <NUREGO_API_URL>${NUREGO_API_URL}</NUREGO_API_URL>
                                 <ZAC_UAA_URL>${ZAC_UAA_URL}</ZAC_UAA_URL>
-                                <ZAC_URL>${ZAC_URL}</ZAC_URL>
-                                <!-- ACS zones (a.k.a tenancies) used for the tests -->
-                                <ZONE1_NAME>${acs.zone.subdomain}</ZONE1_NAME>
-                                <ZONE1_UAA_URL>${ZONE1_UAA_URL}</ZONE1_UAA_URL>
-                                <ZONE2_NAME>${acs.zone.subdomain}2</ZONE2_NAME>
-                                <ZONE2_UAA_URL>${ZONE2_UAA_URL}</ZONE2_UAA_URL>
-                                <ZONE3_NAME>${acs.zone.subdomain}3</ZONE3_NAME>
-                                <!-- proxy settings -->
-                                <http.nonProxyHosts>${NON_PROXY_HOSTS}</http.nonProxyHosts>
-                                <https.proxyHost>${HTTPS_PROXY_HOST}</https.proxyHost>
-                                <https.proxyPort>${HTTPS_PROXY_PORT}</https.proxyPort>
+                                <ZAC_URL>http://localhost:8888</ZAC_URL>
```

Comment:
This can be removed since it should pull the `ZAC_URL` property value from the common set of properties defined above